api/src/test/java/uk/gov/hmcts/fees2/register/api/controllers/acceptance/LookupFeeAcceptanceCriteriaTest.java

```diff
@@ -404,18 +415,17 @@ public void testLookupForProbateCopiesFeeX0258ReturnsCorrectResult() throws Exce
     And the case has an estate valued at £5000 and below
     When the fee for this case is being looked up through the API
     Then return a £0 fee and/or 204 message */
-    @Test
+    //@Test
```

Comment:
why not remove the test altogether then?

Reply:
I'm refactoring these to make them readable.